### PR TITLE
link to release and close button

### PR DESCRIPTION
**Pull request by o-mehlii**

### CODE REVIEW

File: electron-src/index.ts [hunk 1]
```diff
@@ -119,12 +119,8 @@ ipcMain.on("dictionaty-update", (event, word: string) => {
   mainWindow?.webContents.session.addWordToSpellCheckerDictionary(word);
 });
 
-ipcMain.on("slack-redirect", (event, isDesktop: boolean) => {
-  shell.openExternal(
-    isDesktop
-      ? "slack://channel?team=T3PV37ANP&id=C069N5LUP3M"
-      : "https://ukad.slack.com/archives/C069N5LUP3M"
-  );
+ipcMain.on("redirect", (event, link: string) => {
```
Comment (okhmelevskyi) [inline electron-src/index.ts:122]: If you don't use "event" in your logic then replace it with _.

File: renderer/src/components/ActivitiesSection/ActivitiesSection.tsx [hunk 2]
```diff
@@ -98,6 +107,17 @@ export default function ActivitiesSection({
     };
   }, []);
 
+  const updateDownloadClickHandler = () => {
+    global.ipcRenderer.send(
+      "redirect",
```
Comment (okhmelevskyi) [inline renderer/src/components/ActivitiesSection/ActivitiesSection.tsx:112]: I guess, It'd be nice to use a constant for 'redirect' word and replace it everywhere, where is used.

File: renderer/src/components/UpdateDescription/UpdateDescription.tsx [hunk 3]
```diff
@@ -18,6 +18,8 @@ export default function UpdateDescription() {
   );
 
   const [isOpen, setIsOpen] = useState(update?.age === "new");
+  const slackDesktopLink = "slack://channel?team=T3PV37ANP&id=C069N5LUP3M";
```
Comment (okhmelevskyi) [inline renderer/src/components/UpdateDescription/UpdateDescription.tsx:21]: You can move these two constants to another file named constants.ts and import them from it. 
Example: SLACK_DESTOP_LINK = "slack://channel?team=T3PV37ANP&id=C069N5LUP3M";
And the same for another.

**Review by a-nahaitsev — APPROVED** [review]

File: electron-src/helpers/constants.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+export const ipcMainChannels = {
```
Comment (okhmelevskyi) [inline electron-src/helpers/constants.ts:1]: Could you please make the names of constants in upper case? 
Examples:
IPC_MAIN_CHANNELS = {}
REDIRECT: 'redirect'
GET_CURRENT_VERSION: get'CurrentVersion'

**Review by okhmelevskyi — APPROVED** [review]

**Pull request merged**